msteams-platform/concepts/resource-specific-consent.md

```diff
@@ -0,0 +1,90 @@
+
+![Consent screen.](../../assets/images/rsc/rsc-consentscreen.png)
+
+## Prerequisites (Admin)
+
+As an admin, you can control whether team owners in your organization can grant team-specific consent (enabled by default) through [settings](https://review.docs.microsoft.com/en-us/MicrosoftTeams/resource-specific-consent?branch=v-lanac-rsc) that you configure in the Azure Active Directory (Azure AD) PowerShell module or the Azure portal and the Microsoft Teams admin center.
+
+
+> **Important** : Changing any of these settings doesn't affect data access for apps that were already granted consent. 
+
+## Configure Teams App to support RSC (Developer)
+
+The basic steps required to configure a service and get a token from the Microsoft identity platform endpoint that your service can use to call Microsoft Graph are:
+
+1. Register your application in Azure AD.
+2. Validate permissions.
+3. Update your Teams app manifest to link to your AAD app ID.
+4. Get an access token.
+5. Make a Graph call.
+
+
+#### 1. Register a new application in Azure AD
+
+[Register](~/graph/auth-register-app-v2) a new application in the [Azure Active Directory- App Registration portal](https://ms.portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade) to get an AAD app ID. If your AAD app supports [single sign-on (SSO) for tabs](~/microsoftteams/platform/tabs/how-to/authentication/auth-aad-sso), then use the same AAD app ID for configuring RSC.
+
+#### 2. Validate permissions
+* If you are configuring RSC with a newly registered app (with default permissions): Follow step 3. 
+* If you are configuring RSC with an existing app (with additional permissions): Check the permissions in the `API Permissions` section of the app registration portal in Azure AD, and confirm if RSC related permissions are configured here. If so, they should be deleted from here and are supposed to be configured in the Teams app manifest as mentioned in step 3.
+
+#### 3. Update your Teams app manifest to link to your AAD app ID
+Add a [webApplicationInfo](~/microsoftteams/platform/resources/schema/manifest-schema#webapplicationinfo) section to the manifest that includes two properties - `id` is the AAD app ID, and `applicationPermissions` are the RSC permissions (scopes) your app requires.
+
+> **Note**: If your app needs non RSC permissions, you can continue to use them as always, but they do not go in the manifest. 
+
+```
+{ 
+  "$schema": ".../MicrosoftTeams.schema.json", 
+   "webApplicationInfo": { 
+      "id": "cb38cf54-ac89-4a7a-9ea3-095d3d080037", 
+      "applicationPermissions": [  
+      "File.Read.Group",
+      "TeamSettings.Read.Group",
+      "ChannelMessage.Read.Group",
+      "Member.Read.Group"
+      ] 
+  }
+  ... 
+} 
+```
+Refer to the list of available Graph permissions for RSC in [Teams permissions](~/graph/permissions-reference.md) section.
```

Comment:
Link directly to the appropriate section using a # anchor

Reply:
Done